markdown/en/ff-searchbox.md

```diff
@@ -1,9 +1,18 @@
 ## Setup the communication
-First, add the `ff-communication` tag to every page that uses the **FACT-Finder Webcomponents** . With this element you define the location of your FACT-Finder Server in the `url` attribute and which `channel` you want to use in the channel attribute.
+First, add the `ff-communication` tag to every page that uses
+**FACT-Finder Webcomponents**. This element is, in part, used to define
+the location of your FACT-Finder Server using the `url` attribute and
+the `channel` you want to use using the `channel` attribute.
 
-The `ff-communication` Tag has more attributes for example the `default-query` attribute to define a Searchterm whis is always used on pageload. When using the `search-immediate` attribute, which requires no parameter, the first search query is fired once the page has finished loading.
+The `ff-communication` tag has more attributes. One example is the
+`default-query` attribute used to define a default search term that is
+always used on pageload. When using the `search-immediate` attribute,
+which requires no parameter, the search query is fired once the page has
+finished loading.
 
-For more information, see the [API reference](http://web-components.fact-finder.de/documentation/ff-searchbox#communication-api) on the bottom of this page.
+The following code-example shows the aforementioned settings.
+
```

Comment:
Move _For more information, see the [API reference](/api/ff-searchbox#tab=api)._ below the example, so that after _The following code-example shows the aforementioned settings._ really the code example follows